### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by HigorAlves** [dracula/dracula-ui#60] [Ideas] [answered]
Hey guys, what about creating a roadmap (without fixed dates) for just we all know about what is going to enter in new versions or what is being prepared?

**Comment by zenorocha** [comment, marked as answer]
Great idea @HigorAlves!

**Comment by synmux** [comment]
+1, this would be really helpful for deciding whether to integrate Dracula in the short or long term.

**Comment by jaojpaulo** [comment]
Yeah, it would be really nice!

**Comment by redbrain** [comment]
+1 here
